index.php

```diff
@@ -8,7 +16,6 @@
   <link rel="stylesheet" type="text/css" href="style.css">
   </head>
   <body>
-  <h1>Hej TF</h1>
-
+      <h1>Hej TF</h1>
```

Comment:
Maybe take away these changes? I think they might be the cause of the merge conflict.